File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/entity/trip_response/Request.java
package com.zuk0.gaijinsmash.riderz.data.local.entity.trip_response;

import org.simpleframework.xml.ElementList;

import java.util.List;

public class Request {

    @ElementList(inline = true)
    private List<Trip> tripList;

    public List<Trip> getTripList() {
        return tripList;
    }
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/entity/trip_response/Request.kt
package com.zuk0.gaijinsmash.riderz.data.local.entity.trip_response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Request {
    @SerializedName("trip")
    @Expose
    var tripList: List<Trip>? = null

}